src/main/java/roomescape/Controller/ReservationController.java

```diff
@@ -0,0 +1,42 @@
+package roomescape.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.DTO.ReservationRequest;
+import roomescape.DTO.ReservationResponse;
+import roomescape.Service.ReservationService;
+
+import java.util.List;
+
+@RestController
+public class ReservationController {
+    private final String PATH = "reservations";
+    private final ReservationService reservationService;
+
+    @Autowired
+    public ReservationController(ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
+
+    @GetMapping(PATH)
+    public ResponseEntity<List<ReservationResponse>> read() {
+        List<ReservationResponse> reservations = reservationService.findAll();
+        return ResponseEntity.status(HttpStatus.OK).body(reservations);
```

Comment:
```suggestion
        return ResponseEntity.ok().body(reservations);
```
ResponseEntity 에서 제공해주는 정적 메서드를 활용해도 좋을 것 같네요 :)

Reply:
ResponseEntity 가 모든 상태코드에 대한 정적 메서드를 제공해주지 않는 것 같아서, 통일성을 주기 위해 status 정적 메서드를 활용해서 구현헀습니다!
그렇지만 구현되어있는 상태코드에 한해서는 정적 메서드를 사용하는 게 좋을까요??

Reply:
통일성 측면으로 보면 그럴수도 있네요 👍 정답은 없으니 예은님이 보시기 편한대로 구현하면 좋을 것 같네요 !